superset-frontend/src/explore/controlUtils/getControlState.ts

```diff
@@ -90,7 +85,7 @@ export function applyMapStateToPropsToControl<T = ControlType>(
   const { mapStateToProps } = controlState;
   let state = { ...controlState };
   let { value } = state; // value is current user-input value
-  if (mapStateToProps && controlPanelState) {
```

Comment:
we seem to be checking for truthiness of `controlPanelState` here and in a few other places, despite the signature stating it will always be truthy.